src/CommunityVoices/App/Api/Component/SecureContainer.php

```diff
@@ -24,15 +24,17 @@ public function contain($decoratedInstance)
             $user = $this->identifier->identify();
 
             if (!is_object($contained)) {
+                // This should be more verbose, but we cannot print out a class name
+                // of a non-object. So, what to do here?
                 throw new \Exception('No decorated instance!');
```

Comment:
Maybe 'A secure container must have a container object' or something?

Reply:
i would not question that -- that sounds like a good idea.

Reply:
Would that class file go in the same folder then?

Reply:
Creating an empty class that extends `\Exception` accomplishes the goal I had in mind (see https://github.com/EnvironmentalDashboard/community-voices/commit/aa67f915dd384f67114aa8e753a90b172fb24f1b).  Now, it says uncaught `SecureContainerException` which I think is good.  Do you see any reason to extend its behavior any further at all?